apps/web/src/app/[locale]/challenge/_components/comments/comment.tsx

```diff
@@ -225,16 +255,39 @@ function SingleComment({
     }
   }
 
-  async function copyCommentUrlToClipboard() {
-    await navigator.clipboard.writeText(`${window.location.href}/${comment.id}`);
+  async function copyCommentUrlToClipboard(isReply: boolean) {
+    const commentId = isReply ? comment.parentId : comment.id;
+    await navigator.clipboard.writeText(
```

Comment:
will this work if the user:
copies a comment url
goes to that url
copies another comment url?

see: https://github.com/Rand0mLetters/typehero/blob/8926155f945427da69aff0a5a58b5976e18cebea/apps/web/src/app/%5Blocale%5D/challenge/_components/comments/comment.tsx#L294-L311

Reply:
yes cause its using origin